tmt/steps/prepare/ansible.py

```diff
@@ -54,20 +56,20 @@ def options(cls, how=None):
                 help='Optional arguments for ansible-playbook.')
             ] + super().options(how)
 
-    def default(self, option, default=None):
+    def default(self, option: str, default: Optional[Any] = None) -> Any:
         """ Return default data for given option """
         if option == 'playbook':
             return []
         return default
 
-    def wake(self, keys=None):
+    def wake(self, keys: Optional[List[str]] = None):
         """ Wake up the plugin, process data, apply options """
         super().wake(keys=keys)
 
         # Convert to list if necessary
         tmt.utils.listify(self.data, keys=['playbook'])
 
-    def go(self, guest):
+    def go(self, guest: 'GuestArtemis') -> None:
```

Comment:
`tmt.steps.provision.Guest` will be better choice, `GuestArtemis` is a subclass specific for a particular provision plugin.